packages/arb-token-bridge-ui/src/components/TransferPanel/EstimatedGas.tsx

```diff
@@ -136,7 +147,9 @@ export function EstimatedGas({
         >
           <span className="text-right">
             {formatAmount(estimatedGasFee, {
-              symbol: nativeCurrency.symbol
+              symbol: isParentChain
+                ? parentChainNativeCurrency.symbol
+                : childChainNativeCurrency.symbol
```

Comment:
That would be the same as `sourceChainNativeCurrency.symbol` no?

Reply:
no because this component is used on both source and destination boxes